command/server.go

```diff
@@ -2623,6 +2624,11 @@ func startHttpServers(c *ServerCommand, core *vault.Core, config *server.Config,
 		if ln.Config == nil {
 			return fmt.Errorf("Found nil listener config after parsing")
 		}
+
```

Comment:
Ditto this one.  There is a whitespace difference that will have a conflict when merging oss->ent.